caffe2/python/memonger.py

```diff
@@ -798,15 +798,29 @@ def canonical_name(blob):
             op.output[i] = canonical_name(output)
 
 
-
 def apply_recurrent_blob_assignments(op, blob_assignments, canonical_name):
     log.debug("Applying assignments to recurrent op: {}".format(op.type))
+
+    # Apply on alias_dst
+    alias_dst_args = [a for a in op.arg if a.name.endswith("alias_dst")]
+    for alias_dst in alias_dst_args:
+        for i, blob in enumerate(alias_dst.strings):
+            alias_dst.strings[i] = canonical_name(blob.decode()).encode();
```

Comment:
Wow, why semicolon at the end of the satement?

Reply:
Aha! Something a C++ guy cannot resist~
No worry, let me sent a quick fix.

Reply:
https://github.com/pytorch/pytorch/pull/74031